tests/infer/mcmc/test_mcmc_util.py

```diff
@@ -0,0 +1,47 @@
+import pytest
+import torch
+
+import pyro
+import pyro.distributions as dist
+from pyro.infer.mcmc import NUTS
+from pyro.infer.mcmc.api import MCMC
+from pyro.infer.mcmc.util import initialize_model, predictive
+from pyro.util import ignore_experimental_warning, optional
+from tests.common import assert_close
+
+
+def beta_bernoulli():
+    N = 1000
+    true_probs = torch.tensor([0.2, 0.3, 0.4, 0.8, 0.5])
+    data = dist.Bernoulli(true_probs).sample([N])
+
+    def model(data=None):
+        with pyro.plate("num_components", 5):
+            beta = pyro.sample("beta", dist.Beta(1., 1.))
+            with pyro.plate("data", N):
+                pyro.sample("obs", dist.Bernoulli(beta), obs=data)
+
+    return model, data, true_probs
+
+
+@pytest.mark.parametrize("num_samples", [100, 200, None])
```

Comment:
It seems that `num_samples` does not play any role here. Could you please revise this test?

Reply:
Thanks for catching this; fixed!